### ISSUE TITLE
Question about the weight in multi-loss function

### ISSUE THREAD

**Issue by bellhyeon** [hnuzhy/BPJDet#20]
Hi, I have a question about your great work.

How did you determine the lambda value(Weight of the BPD Loss; 0.025 in hyp)?

Thank you in advance for the response!

**Comment by hnuzhy** [comment]
Thank you for your recognition of this work.

This question involves some techniques for adjusting multi-loss parameters. Usually, my experience is to keep the values of multiple losses at the same magnitude. For example, in original YOLOv5 for object detection, it has three kinds of losses (`cls`, `box`, and `obj`). There values are adjusted into a rough range (0.01, 0.1). Thus, we also tried to adjust the newly added body-part association loss `bpb` into this range. We finally locked the magnitude of its weight `\lambda$` at `1e-2`. And we searched the optimal result of `\lambda$` by ablation studies. Details can be found in [our paper](https://arxiv.org/abs/2304.10765) (see Section 5.2.1)

Below is the training loss WANDB-logs of ablation studies about Loss Weight `\lambda$`

![image](https://github.com/hnuzhy/BPJDet/assets/21312423/a9234c4d-7012-4c10-8506-183a75be60f6)

Hope this will be helpful to you!
